android/src/main/java/com/horcrux/svg/SvgPackage.java

```diff
@@ -10,50 +10,134 @@
+      specs.put(TextViewManager.REACT_CLASS, ModuleSpec.viewManagerSpec(() -> new TextViewManager()));
+      specs.put(TSpanViewManager.REACT_CLASS, ModuleSpec.viewManagerSpec(() -> new TSpanViewManager()));
+      specs.put(TextPathViewManager.REACT_CLASS, ModuleSpec.viewManagerSpec(() -> new TextPathViewManager()));
+      specs.put(ImageViewManager.REACT_CLASS, ModuleSpec.viewManagerSpec(() -> new ImageViewManager()));
+      specs.put(ClipPathViewManager.REACT_CLASS, ModuleSpec.viewManagerSpec(() -> new ClipPathViewManager()));
+      specs.put(DefsViewManager.REACT_CLASS, ModuleSpec.viewManagerSpec(() -> new DefsViewManager()));
+      specs.put(UseViewManager.REACT_CLASS, ModuleSpec.viewManagerSpec(() -> new UseViewManager()));
+      specs.put(SymbolManager.REACT_CLASS, ModuleSpec.viewManagerSpec(() -> new SymbolManager()));
+      specs.put(LinearGradientManager.REACT_CLASS, ModuleSpec.viewManagerSpec(() -> new LinearGradientManager()));
+      specs.put(RadialGradientManager.REACT_CLASS, ModuleSpec.viewManagerSpec(() -> new RadialGradientManager()));
+      specs.put(PatternManager.REACT_CLASS, ModuleSpec.viewManagerSpec(() -> new PatternManager()));
+      specs.put(MaskManager.REACT_CLASS, ModuleSpec.viewManagerSpec(() -> new MaskManager()));
+      specs.put(ForeignObjectManager.REACT_CLASS, ModuleSpec.viewManagerSpec(() -> new ForeignObjectManager()));
+      specs.put(MarkerManager.REACT_CLASS, ModuleSpec.viewManagerSpec(() -> new MarkerManager()));
+      specs.put(SvgViewManager.REACT_CLASS, ModuleSpec.viewManagerSpec(() -> new SvgViewManager()));
+      mViewManagers = specs;
+    }
+    return mViewManagers;
+  }
+
+  /** {@inheritDoc} */
+  @Override
+  public Collection<String> getViewManagerNames(ReactApplicationContext reactContext) {
+    return getViewManagersMap(reactContext).keySet();
+  }
+
+  /** {@inheritDoc} */
   @Override
-  public List<ViewManager> createViewManagers(@Nonnull ReactApplicationContext reactContext) {
-    return Arrays.<ViewManager>asList(
-        new GroupViewManager(),
-        new PathViewManager(),
-        new CircleViewManager(),
-        new EllipseViewManager(),
-        new LineViewManager(),
-        new RectViewManager(),
-        new TextViewManager(),
-        new TSpanViewManager(),
-        new TextPathViewManager(),
-        new ImageViewManager(),
-        new ClipPathViewManager(),
-        new DefsViewManager(),
-        new UseViewManager(),
-        new SymbolManager(),
-        new LinearGradientManager(),
-        new RadialGradientManager(),
-        new PatternManager(),
-        new MaskManager(),
-        new ForeignObjectManager(),
-        new MarkerManager(),
-        new SvgViewManager());
+  public @Nullable ViewManager createViewManager(
+    ReactApplicationContext reactContext, String viewManagerName) {
+    ModuleSpec spec = getViewManagersMap(reactContext).get(viewManagerName);
+    return spec != null ? (ViewManager) spec.getProvider().get() : null;
   }
```

Comment:
Let's implement one more method:

```
  @Override
  public List<ModuleSpec> getViewManagers(ReactApplicationContext reactContext) {
    return new ArrayList<>(getViewManagersMap(reactContext).values());
  }
```

I think this should fix our `requireNativeComponent: "RNSVGCircle" was not found in the UIManager` problem.